pysteps/utils/reprojection.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+"""
+pysteps.utils.reprojection
+==================
+
+Reprojection tool to reproject the grid and adjust the grid cell size of an 
+input field to a destination field.
+
+.. autosummary::
+    :toctree: ../generated/
+
+    reprojection
+"""
+from pysteps.exceptions import MissingOptionalDependency
+import xarray as xr
+import numpy as np
+
+try:
+    import rasterio
+
+    RASTERIO_IMPORTED = True
+except ImportError:
+    RASTERIO_IMPORTED = False
+from rasterio import Affine as A
+from rasterio.warp import reproject, Resampling
```

Comment:
these two import will throw an error if rasterio is not installed, you can include them in the try ... except block, too